うーん、うまくいかないようです。 何か見落していますか？

```console
[root@rinux car]# uname -a
Linux rinux.local.kusaremkn.com 6.8.12-5-pve #1 SMP PREEMPT_DYNAMIC PMX 6.8.12-5 (2024-12-03T10:26Z) x86_64 x86_64 x86_64 GNU/Linux
[root@rinux car]# cat /etc/os-release 
NAME="AlmaLinux"
VERSION="9.6 (Sage Margay)"
ID="almalinux"
ID_LIKE="rhel centos fedora"
VERSION_ID="9.6"
PLATFORM_ID="platform:el9"
PRETTY_NAME="AlmaLinux 9.6 (Sage Margay)"
ANSI_COLOR="0;34"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:almalinux:almalinux:9::baseos"
HOME_URL="https://almalinux.org/"
DOCUMENTATION_URL="https://wiki.almalinux.org/"
BUG_REPORT_URL="https://bugs.almalinux.org/"

ALMALINUX_MANTISBT_PROJECT="AlmaLinux-9"
ALMALINUX_MANTISBT_PROJECT_VERSION="9.6"
REDHAT_SUPPORT_PRODUCT="AlmaLinux"
REDHAT_SUPPORT_PRODUCT_VERSION="9.6"
SUPPORT_END=2032-06-01
[root@rinux car]# make rpm
cc -O -Wall -o car car.c -lncurses
# * rpmbuild --sign requires 'rpm-sign' package installed.
echo pwd=
pwd=
export SRC_ROOT=/root/car && rpmbuild -D "_topdir /root/car/rpmbuild" -D "_commit_id 0300503" -ba car.spec --sign
error: rpmbuild --sign is no longer supported. Use the rpmsign command instead!
make: *** [Makefile:24: rpm] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -19,6 +19,8 @@ car: car.c car.h
 	$(CC) $(CFLAGS) -o car car.c -lncurses
 
 rpm: car car.spec
+	# To install dependenciesm run:
+	#   dnf install install -y git make gcc ncurses-devel rpm-build
 	echo pwd=$(pwd)
 	export SRC_ROOT=$(shell pwd) && $(RPMBUILD) -D "_topdir $(shell pwd)/rpmbuild" -D "_commit_id $(shell git log -1 --oneline|cut -b -7)" -ba car.spec
 

--- a/Makefile
+++ b/Makefile
@@ -19,9 +19,8 @@ car: car.c car.h
 	$(CC) $(CFLAGS) -o car car.c -lncurses
 
 rpm: car car.spec
-	# * rpmbuild --sign requires 'rpm-sign' package installed.
 	echo pwd=$(pwd)
-	export SRC_ROOT=$(shell pwd) && $(RPMBUILD) -D "_topdir $(shell pwd)/rpmbuild" -D "_commit_id $(shell git log -1 --oneline|cut -b -7)" -ba car.spec --sign
+	export SRC_ROOT=$(shell pwd) && $(RPMBUILD) -D "_topdir $(shell pwd)/rpmbuild" -D "_commit_id $(shell git log -1 --oneline|cut -b -7)" -ba car.spec
 
 clean:
 	rm -f car

--- a/car.spec
+++ b/car.spec
@@ -40,6 +40,7 @@ Original Copyright: sl Copyright 1993,1998,2014 Toyoda Masashi (mtoyoda@acm.org)
 mkdir -p %{buildroot}/usr/bin/
 mkdir -p %{buildroot}/usr/share/man/man1
 mkdir -p %{buildroot}/usr/share/man/ja/man1
+mkdir -p %{_sourcedir}
 
 #echo "SRC_ROOT = $SRC_ROOT"
 cp $SRC_ROOT/car %{buildroot}/usr/bin

